WolvenKit.App/ViewModels/HomePage/Pages/ModsViewModel.cs

```diff
@@ -0,0 +1,283 @@
+                    "RedMod not found");
+
+                switch (res)
+                {
+                    case WMessageBoxResult.OK:
+                    case WMessageBoxResult.Yes:
+                        _homePageViewModel.NavigateTo(EHomePage.Plugins);
+                        break;
+                }
+            }
+        }
+
+        public ICommand RemoveCommand { get; private set; }
+        private void Remove()
+        {
+            if (SelectedMod is null)
+            {
+                return;
+            }
+
+            if (!Directory.Exists(SelectedMod.Path))
+            {
+                _logger.Warning($"Could not find mod: {SelectedMod.Path}");
+            }
+
+            try
+            {
+                Directory.Delete(SelectedMod.Path, true);
+
+                Mods.Remove(SelectedMod);
+            }
+            catch (Exception)
+            {
+                _logger.Error($"Could not delete mod: {SelectedMod.Path}");
+            }
+        }
+
+        #endregion
+
+        private void LoadMods()
+        {
+            LoadModsInfo();
+
+            // parse existing mods.info and update enabled
+            // also update load order
+            var modsInfoPath = Path.Combine(_settings.GetRED4GameRootDir(), "r6", "cache", "modded", "mods.json");
+            ModsInfo modsInfo = null;
+            var foundMods = new List<ModInfoViewModel>();
+            if (File.Exists(modsInfoPath))
+            {
+                try
+                {
+                    modsInfo = JsonSerializer.Deserialize<ModsInfo>(File.ReadAllText(modsInfoPath), _options);
+                }
+                catch (Exception)
+                {
+                    _logger.Warning($"Could not read mods info file: {modsInfoPath}");
+                    return;
+                }
+
+
+                for (var i = 0; i < modsInfo.Mods.Count; i++)
+                {
+                    var mod = modsInfo.Mods[i];
+
+                    var local = Mods.FirstOrDefault(x => x.Folder == mod.folder);
+                    if (local is not null)
+                    {
+                        local.IsEnabled = mod.enabled;
+                        local.LoadOrder = i;
+                        foundMods.Add(local);
+                    }
+                }
+            }
+
+            // loop through all existing mods
+            // reorder according to modsInfo
+            // first come all the mods in mods.info
+            // then the rest
+            var rest = Mods.Where(x => !foundMods.Contains(x)).ToList();
+            for (var i = 0; i < rest.Count; i++)
+            {
+                var mod = rest[i];
+                mod.LoadOrder = i + foundMods.Count;
+            }
+
+            _logger.Info($"Found {Mods.Count} mods.");
+
+        }
+
+        private void LoadModsInfo()
```

Comment:
I would create an array / List / typeof(Mods) which I return or accept it as a ref parameter. I don't like doing this on globals, then modify them in the callee.

```suggestion
        private typeof(Mods) LoadModsInfo() // Actual type
```

or

```suggestion
        private void LoadModsInfo(ref typeof(Mods) mods) // Actual type
```